spec/changelog.yml

```diff
@@ -24,9 +24,9 @@
     link: https://github.com/elastic/package-spec/pull/607
 - version: 2.12.1-next
   changes:
-  - description: Prepare for next version
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/612
+  - description: Fix validation of filters in built dashboards
+    type: bugfix
+    link: https://github.com/elastic/package-spec/pull/613
```

Comment:
As this is for v3.0.0 , should it be moved to the 3.0.0-next block ?
In v2.12 or previous versions, that rule is not checked.

Reply:
You are right, and as this hasn't been released yet I won't add any changelog entry. Thanks!